pkg/v1/resource/chart/resource.go

```diff
@@ -0,0 +1,134 @@
+package chart
+
+import (
+	"reflect"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/apprclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/cluster-operator/pkg/cluster"
+	"github.com/giantswarm/cluster-operator/pkg/v1/guestcluster"
+	"github.com/giantswarm/cluster-operator/pkg/v1/key"
+)
+
+const (
+	// Name is the identifier of the resource.
+	Name = "chartv1"
+
+	chartOperatorChart   = "chart-operator-chart"
+	chartOperatorChannel = "0.1-beta"
```

Comment:
When we need to update chart-operator we will update the channel here. This will be versioned via the cluster-operator version bundle.